zhaquirks/smartthings/multiv4.py

```diff
@@ -0,0 +1,59 @@
+from zigpy.profiles import PROFILES, zha
+from zigpy.zcl.clusters.general import Basic, Identify,\
+    PollControl, Ota, BinaryInput
+from zigpy.zcl.clusters.security import IasZone
+from zhaquirks.centralite import PowerConfigurationCluster
+from zigpy.zcl.clusters.measurement import TemperatureMeasurement
+from zigpy.quirks import CustomDevice
+
+
+class SmartThingsMultiV4(CustomDevice):
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+    signature = {
+        # <SimpleDescriptor endpoint=1 profile=260 device_type=1026
+        # device_version=0
+        # input_clusters=[0, 1, 3, 15, 32, 1026, 1280, 64514]
+        # output_clusters=[25]>
+        1: {
```

Comment:
we match on man / model now so add them here

Reply:
Done